templateflow/tests/test_api.py

```diff
@@ -24,7 +24,7 @@
     "\tauthor = {Vladimir Fonov and Alan C. Evans and Kelly Botteron and C. "
     "Robert Almli and Robert C. McKinstry and D. Louis Collins},\n"
     "\ttitle = {Unbiased average age-appropriate atlases for pediatric studies},\n"
-    "\tjournal = {{NeuroImage}}\n}"
+    "\tjournal = {NeuroImage}}\n}"
```

Comment:
why this?

Reply:
i believe it was a bug in earlier versions of `doi2bib` (fixes tests when using 0.4)

Reply:
seems imbalanced - the other examples don't have the second closing curly brace

Reply:
opened an issue at https://github.com/bibcure/doi2bib/issues/6